### PR TITLE
Corrected Azure web app version

### PR DESCRIPTION
**Pull request by davenowell**
This PR refers to issue #18

### CODE REVIEW

**Comment by brianamarie** [comment]
✋ I will test this now!

**Comment by brianamarie** [comment]
Edited my comment - I am going to take my ring out of the hat for reviewing at this time. Sorry for the false alarm!

**Comment by hectorsector** [comment]
I attached the source issue #18 to the project board and into the [needs review column](https://github.com/orgs/githubtraining/projects/1#column-10192932). This should help additional folks see that it needs review.

**Comment by lmkeston** [comment]
> 
> 
> ~✋ I will test this now!~ I take this back - I will not test this now. Spinning up the Azure environment would take me a long time because I haven't done it in a long time. Maybe someone who is closer to the Azure space or to this course would be better to test, like @lmkeston or @mattdavis0351? Or anyone else who has the bandwidth to get it going 😊

I can run through this @brianamarie.  I'll assign it to me on the board. 

cc: @hectorsector @davenowell

**Comment by hectorsector** [comment]
@lmkeston the newest deployment should be functional and you should now be able to test the course

**Comment by davenowell** [comment]
@hectorsector So, clicking view deployment after your change gives me a 404 page.  What am I doing wrong?

**Comment by hectorsector** [comment]
I have 3 potential solutions:
1. Make sure you're signed into Learning Lab
2. Make sure your staff bar is enabled on Learning Lab
3. Make sure you're clicking on the most recent Deployment button

If all those 3 are confirmed maybe it's something to do with permissions and we have to look deeper.

**Comment by lmkeston** [comment]
> 
> 
> @lmkeston the newest deployment should be functional and you should now be able to test the course

@hectorsector, it's letting me test now - thanks!

**Comment by mickeygousset** [comment]
Step 7 now fails for a different reason.  It can't find the action.

image

**Comment by mickeygousset** [comment]
There is no V2 version of Azure/webapps-container-deploy. The latest is v1.0.1

**Comment by mickeygousset** [comment]
ooohhhh, we have to use a completely different action it seems: ##[warning]Input 'app-name' has been deprecated with message: This action is ARCHIVED and will not receive any updates, update your workflows to use azure/webapps-deploy@v2

Going to try that.

**Comment by mickeygousset** [comment]
Ok, that is the problem.   In the deploy_staging.yml file, we have to change the azure/web-container-deploy@v2 to azure/webapps-deploy@v2

File: responses/03_workflow-steps.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ jobs:
           password: {% raw %}${{ secrets.GITHUB_TOKEN }}{% endraw %}
 
       - name: Deploy web app container
-        uses: azure/webapps-container-deploy@v1
+        uses: azure/webapps-container-deploy@v2
```
Comment (mickeygousset) [inline responses/03_workflow-steps.md:144]: This is using the wrong action.  This action has been deprecated. This line should be changed to:

uses: azure/webapps-deploy@v2

Everything else can stay the same

**Pull request merged**